modules/server/src/main/resources/Tcs.xml

```diff
@@ -1800,56 +1810,6 @@
             <type>STRING</type>
             <description>P2 Object Name</description>
         </attribute>
-        <attribute name="p2ara">
```

Comment:
All these definitions are used to set the P2 target keywords! They must be restored.